src/utransport.rs

```diff
@@ -593,18 +583,20 @@ impl UTransport for UPClientZenoh {
             // This is special UUri which means we need to unregister both listeners
```

Comment:
```rust
// This is special UUri which means we need to unregister both listeners
```
=>
```rust
// This is special UUri which means we need to unregister all listeners
```
?

Reply:
Thank you! The legacy comments I didn't notice before. 😆